Index: syslinux-3.84/Makefile
===================================================================
--- syslinux-3.84/Makefile
+++ syslinux-3.84/Makefile	2009-12-22 15:26:29.000000000 +0100
@@ -42,7 +42,7 @@
 	mbr/mbr_c.bin mbr/altmbr_c.bin mbr/gptmbr_c.bin \
 	mbr/mbr_f.bin mbr/altmbr_f.bin mbr/gptmbr_f.bin \
 	core/pxelinux.0 core/isolinux.bin core/isolinux-debug.bin \
-	gpxe/gpxelinux.0 dos/syslinux.com win32/syslinux.exe \
+	gpxe/gpxelinux.0 dos/syslinux.com \
 	dosutil/*.com dosutil/*.sys \
 	$(MODULES)
 
@@ -53,7 +53,7 @@
 # files that depend only on the B phase, but may have to be regenerated
 # for "make installer".
 BSUBDIRS = codepage core memdisk modules com32 mbr memdump gpxe sample \
-	   libinstaller dos win32 dosutil
+	   libinstaller dos dosutil
 ITARGET  =
 IOBJECTS = $(ITARGET) dos/copybs.com \
 	utils/gethostip utils/isohybrid utils/mkdiskimage \
@@ -67,9 +67,9 @@
 # Things to install in /usr/lib/syslinux
 INSTALL_AUX   =	core/pxelinux.0 gpxe/gpxelinux.0 core/isolinux.bin \
 		core/isolinux-debug.bin \
-		dos/syslinux.com dos/copybs.com win32/syslinux.exe \
+		dos/syslinux.com dos/copybs.com \
 		mbr/*.bin $(MODULES)
-INSTALL_AUX_OPT = win32/syslinux.exe
+INSTALL_AUX_OPT = 
 
 # These directories manage their own installables
 INSTALLSUBDIRS = com32 utils dosutil
