diff -Nur syslinux-4.05.orig/Makefile syslinux-4.05/Makefile
--- syslinux-4.05.orig/Makefile	2011-12-09 22:28:17.000000000 +0400
+++ syslinux-4.05/Makefile	2012-09-28 14:19:18.000000000 +0400
@@ -29,7 +29,7 @@
 # with their own Makefiles.  Finally, there is a list of those
 # directories.
 #
-
+LD              = $(CROSS_COMPILE)ld.bfd
 # List of module objects that should be installed for all derivatives
 MODULES = memdisk/memdisk memdump/memdump.com modules/*.com \
 	com32/menu/*.c32 com32/modules/*.c32 com32/mboot/*.c32 \
diff -Nur syslinux-4.05.orig/mk/syslinux.mk syslinux-4.05/mk/syslinux.mk
--- syslinux-4.05.orig/mk/syslinux.mk	2011-12-09 22:28:17.000000000 +0400
+++ syslinux-4.05/mk/syslinux.mk	2012-09-28 14:24:32.000000000 +0400
@@ -47,7 +47,7 @@
 		   then echo '$(1)'; else echo '$(2)'; fi; \
 		   rm -f $$tmpf)
 
-LD	 = ld
+LD	 = ld.bfd
 OBJDUMP	 = objdump
 OBJCOPY  = objcopy
 AR       = ar
