--- syslinux-5.01/Makefile.orig	2013-01-29 16:09:55.441557887 +0100
+++ syslinux-5.01/Makefile	2013-01-29 16:10:39.376037673 +0100
@@ -30,6 +30,7 @@ include $(MAKEDIR)/syslinux.mk
 # directories.
 #
 
+LD = $(CROSS_COMPILE)ld.bfd
 MODULES = memdisk/memdisk memdump/memdump.com \
 	com32/menu/*.c32 com32/modules/*.c32 com32/mboot/*.c32 \
 	com32/hdt/*.c32 com32/rosh/*.c32 com32/gfxboot/*.c32 \
--- syslinux-5.01/mk/syslinux.mk.orig	2013-01-29 16:09:55.441557887 +0100
+++ syslinux-5.01/mk/syslinux.mk	2013-01-29 16:10:55.151850196 +0100
@@ -48,7 +48,7 @@ gcc_ok   = $(shell tmpf=gcc_ok.$$$$.tmp;
 		   then echo '$(1)'; else echo '$(2)'; fi; \
 		   rm -f $$tmpf)
 
-LD	 = ld
+LD	 = ld.bfd
 OBJDUMP	 = objdump
 OBJCOPY  = objcopy
 STRIP    = strip
--- syslinux-5.01/utils/Makefile.bero	2013-01-29 16:12:54.454440685 +0100
+++ syslinux-5.01/utils/Makefile	2013-01-29 16:13:10.032256059 +0100
@@ -18,8 +18,8 @@ topdir = ..
 MAKEDIR = $(topdir)/mk
 include $(MAKEDIR)/syslinux.mk
 
-CFLAGS   = $(GCCWARN) -Os -fomit-frame-pointer -D_FILE_OFFSET_BITS=64
-LDFLAGS  = -O2
+CFLAGS   = $(GCCWARN) -Os -fomit-frame-pointer -D_FILE_OFFSET_BITS=64 $(RPM_OPT_FLAGS)
+LDFLAGS  = -O2 $(RPM_OPT_FLAGS) -fuse-ld=bfd
 
 C_TARGETS	 = isohybrid gethostip memdiskfind
 SCRIPT_TARGETS	 = mkdiskimage
